build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.23.0-go1.17.13-bullseye.0
+v1.25.0-go1.19.4-bullseye.0
```

Comment:
you can replace to `v1.23.0-go1.19.4-bullseye.0` :)

Reply:
done